main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
This current setup will only have 8 iterations. Possible edit:

'''
loop = 0
while  loop  < 10:
'''